tests/test_places.py

```diff
@@ -0,0 +1,121 @@
+from apistar.test import TestClient
+from app import app
+import pytest
+import json
+import os
+
+"""
+    This module tests basic query against the endpoint '/places/'
+"""
+
+def load_place(file_name, mimir_client, index_name, _doc_type):
```

Comment:
acn't you guess the index_name by the doc_type ?

Reply:
yup, done